ggml-phi-knc-dot_q5_K_q8_K.c

```diff
@@ -0,0 +1,235 @@
+// For uint32_t
+#include <stdint.h>
+
+// For size_t
+#include <stdio.h>
+
+// Yes, we have to tell this header to actually export stuff.
+#define GGML_COMMON_IMPL_C
+#include "ggml-common.h"
+#include "ggml-quants.h"
+#include "ggml-impl.h"
+
+// FIXME: why do we have to import this twice?
+#define GGML_COMMON_IMPL_C
+// For block_q5_K and block_q8_K. only given the second time.
+#include "ggml-common.h"
```

Comment:
You don't - just remove the one above on lines 7-9

The `ggml-quants.h` brings this include with `GGML_COMMON_DECL_C` for the structs. You only need to to bring it again with `GGML_COMMON_IMPL_C` for the implementation of the kernels

Reply:
thanks!